payjoin/src/receive/v2/session.rs

```diff
@@ -291,11 +293,51 @@ mod tests {
                 },
             }),
         };
-        run_session_history_test(test);
+        run_session_history_test(test)
+    }
+
+    #[test]
+    fn test_replaying_unchecked_proposal_expiry() {
+        let now = SystemTime::now();
+        let session_context = SessionContext {
+            address: SHARED_CONTEXT.address.clone(),
+            directory: SHARED_CONTEXT.directory.clone(),
+            subdirectory: SHARED_CONTEXT.subdirectory.clone(),
+            ohttp_keys: SHARED_CONTEXT.ohttp_keys.clone(),
+            expiry: now,
+            s: SHARED_CONTEXT.s.clone(),
+            e: None,
+        };
```

Comment:
Would the shorthand work?
```suggestion
        let session_context = SessionContext {
    expiry: now,
    e: None,
    ..SHARED_CONTEXT.clone()
};
```